File: result/repair/starchat/python2java/python_repair/MINIMUM_NUMBER_SUBSETS_DISTINCT_ELEMENTS.py
def f_gold(ar, n):
        ar.sort()
        res = 0
        for i in range(n):
            count = 1
            for j in range(i + 1, n):
                if (ar[i]==ar[i+1]):
                    count += 1
                else:
                    break
            res = max(res, count)
        return res

File: result/translate/starchat/java2cpp/c++/MINIMUM_NUMBER_SUBSETS_DISTINCT_ELEMENTS.cpp
int f_gold ( int ar [ ], int n ) {
        int res = 0 ;
        sort(ar, ar+n);
        for (int i = 0; i < n; i++) {
            int count = 1;
            for (int j = i+1; j < n; j++) {
                if (ar[i] == ar[j]) count++;
                else break;
            }
            res = max(res, count);
        }
        return res;
    }